src/server/webkit.ts

```diff
@@ -144,7 +144,7 @@ export class WebKit implements BrowserType<WKBrowser> {
       args = [],
     } = options;
     if (devtools)
-      throw new Error('Option "devtools" is not supported by WebKit');
+      console.warn('devtools parameter is not supported as a launch argument in WebKit. You can launch the devtools window manually.');
```

Comment:
Firefox change looks good, but WebKit is not as debugging-friendly as Chromium and Firefox. In particular, opening Web Inspector will terminate the Playwright connection and I feel like it is worth mentioning it somewhere. We should probably get ourselves the **Debugging ** section in the readme or a full debugging.md where this message would point.

Not a blocker for this change, happy to approve it, mentioning it in case you are interested in taking it further and introducing this debugging tutorial.

Reply:
Would be definitely good to adjust the warning message there. Maybe something like:

`devtools parameter including launching DevTools manually is not supported as a launch argument in WebKit.`

Reply:
It is somewhat nuanced on WebKit - you can open Web Inspector, but it'll terminate the script. Maybe state that explicitly?

Reply:
maybe something like that? (adjusted the PR)
@pavelfeldman